GirafRest/appsettings.template.json

```diff
@@ -24,5 +24,17 @@
         "JwtKey": "<jwt-key>",
         "JwtIssuer": "<jwt-issuer>",
         "JwtExpireDays": 30
+    },
+    "IpRateLimiting": {
+        "EnableEndpointRateLimiting": false,
+        "StackBlockedRequests": false,
+        "HttpStatusCode": 429,
+        "GeneralRules": [
+            {
+                "Endpoint": "*",
+                "Period": "1s",
+                "Limit": 5

```

Comment:
The rate limit of  5 requests for every 1 second will encounter some problems in the app. As some requests for weekplans would require a request to get the week name, year and day. Another request is then needed to get the specific week out from the user id and week year and day.

I.e to get all week for a user with 20 weeks would require
1 request to get week names
20 request to get week

To fix this issue there should be created a new API endpoint for getting weekplans for a user that would only require one request.

Reply:
Talked with PO about this, a new issue regarding this problem has been created. For now, the rate limit is set to 20 as compensation.